src/EnergyPlus/HVACHXAssistedCoolingCoil.cc

```diff
@@ -831,6 +839,10 @@ namespace HVACHXAssistedCoolingCoil {
 			}
 
 			Error = CoilOutputTempLast - Node( HXAssistedCoil( HXAssistedCoilNum ).HXExhaustAirInletNodeNum ).Temp;
+			if ( Iter > 40 ) { // check for oscillation (one of these being negative and one positive) before hitting max iteration limit
```

Comment:
Saw oscillations while testing and added this to avoid max iter limit warnings. Nothing can be done if iterations oscillate back and forth with same result.

Reply:
Did you figure out 40 was a good number?  Or just an educated guess?  I don't have a problem with it, just curious.

Reply:
40 is just a number far enough into the iteration count (max = 50) to let the solution converge before checking if there are oscillations since the solution could easily be positive and negative on successive iterations and still not have a problem converging. I just wanted a few iterations left before checking this.